modules/nextflow/src/test/groovy/nextflow/k8s/model/PodOptionsTest.groovy

```diff
@@ -37,20 +37,20 @@ class PodOptionsTest extends Specification {
         options.getAutomountServiceAccountToken() == true
     }
 
-    def 'should set pullPolicy' () {
+    def 'should set imagePullPolicy' () {
 
         when:
         def options = new PodOptions()
         then:
         options.getImagePullPolicy() == null
         
         when:
-        options = new PodOptions([ [pullPolicy:'Always'] ])
+        options = new PodOptions([ [imagePullPolicy: 'Always'] ])
         then:
         options.getImagePullPolicy() == 'Always'
 
         when:
-        options = new PodOptions([ [imagePullPolicy:'latest'] ])
+        options = new PodOptions([ [imagePullPolicy: 'latest'] ])
         then:
```

Comment:
It should be kept a test also for `pullPolicy` to be sure it's still working

Reply:
Done